### ISSUE TITLE
Blur types in Linux

### ISSUE THREAD

**Issue by keli5** [NyaomiDEV/Glasstron#262]
Going over the readme, it doesn't seem to say which blur type should be used for `blurType`. Should this simply be omitted for Linux platforms? I remember that Glass**cord** only accepted a boolean, but I'm not certain if it's the same to just `setBlur(true)`.

**Comment by KorbsStudio** [comment]
The only blur type supported for Linux is going to be `blurbehind`.

**Comment by keli5** [comment]
Noted, thank you!

**Comment by NyaomiDEV** [comment]
More on this: "Linux platforms" is actually Linux distros using X11 and Plasma -or- Electron applications using XWayland on Plasma Wayland; the blur type there is the blur filter provided by KWin, and we cannot change the intensity as Plasma handles it to be consistent with the entire system. `blurbehind` is the close equivalent on Windows, but you don't need to specify it because it simply doesn't count on Plasma.

Any other support on GNOME is provided by BlurMyShell which is a third party GNOME extension which I did not test myself, not even once; so I cannot comment on it.

**Comment by KorbsStudio** [comment]
BlurMyShell works fine with Glasstron on GNOME. Bugs from Blur Provider are still present sadly.
